treeherder/etl/perf.py

```diff
@@ -127,6 +127,11 @@ def _load_perf_artifact(project_name, reference_data, job_data, job_guid,
                     'framework': framework,
                     'extra_properties': extra_properties,
                     'lower_is_better': subtest.get('lowerIsBetter', True),
+                    'should_alert': subtest.get('shouldAlert'),
```

Comment:
does this mean I will need to modify all instances of talos to include should_alert, etc?  I don't see a default value here.